packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -88,15 +93,35 @@ function BasicColorList({
     .map(getPatternAsString)
     .findIndex((c) => colorAsBackground === c);
 
+  const isLocal = 'local' === colorType;
```

Comment:
Why is it that we use a string `type` and not just bool throughout? It seems overly complicated because we keep converting the string back to a bool as that makes more sense in many places.